org.freedesktop.Sdk.json.in

```diff
@@ -33,6 +33,9 @@
         "--extension=org.freedesktop.Sdk.Extension=subdirectories",
         "--extension=org.freedesktop.Sdk.Extension=directory=lib/sdk",
         "--extension=org.freedesktop.Sdk.Extension=no-autodownload=true",
+        "--extension=org.freedesktop.Platform.VAAPI.Intel=directory=lib/dri/intel-vaapi-driver",
+        "--extension=org.freedesktop.Platform.VAAPI.Intel=version=1.0",
```

Comment:
I think it is better to leave this version out and have the extension version match the runtime version, because the build of the extension is going to be against the 1.6 base libraries, and thus only work for the 1.6 freedesktop runtime (and derivates)